mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -45,6 +45,7 @@ CLASS none/ayr net/minecraft/block/material/Material
 	METHOD <init> (Lnone/ays;)V
 		ARG 0 color
 	METHOD a isFullBlock ()Z
+	METHOD b blocksLight ()Z
```

Comment:
Ah, ok I initially read it as literally 'block' (cube) light, so that's why my suggestion was off. Something like `preventsLightTransmission` perhaps - the inverse of whatever is chosen in response to the following review note. Its strange that Mojang has 2, one in Material and one in Block and one is inverse of the other though it doesn't surprise me...